kolibri/core/content/utils/import_export_content.py

```diff
@@ -97,15 +97,56 @@ def filter_by_file_availability(nodes_to_include, channel_id, drive_id, peer_id)
 
 def get_import_export_data(  # noqa: C901
```

Comment:
Guessing we can probably remove the `noqa` from here now?

Reply:
:+1+ Indeed we can!